docs/quick-tutorial.md

```diff
@@ -201,35 +203,35 @@ thanos store \
 
 ## Compactor
 
-A local Prometheus installation periodically compacts older data to improve query efficiency. Since the sidecar backs up data as soon as possible, we need a way to apply the same process to data in the object storage.
+A local Prometheus installation periodically compacts older data to improve query efficiency. Since the sidecar backs up data into an object storage bucket as soon as possible, we need a way to apply the same process to data in the bucket.
 
-The compactor component simply scans the object storage and processes compaction where required. At the same time it is responsible for creating downsampled copies of data to speed up queries.
+Thanos Compactor simply scans the object storage bucket and performs compaction where required. At the same time, it is responsible for creating downsampled copies of data in order to speed up queries.
 
 ```bash
 thanos compact \
     --data-dir             /var/thanos/compact \  # Temporary workspace for data processing
-    --objstore.config-file bucket_config.yaml \   # Bucket where to apply the compacting
-    --http-address         0.0.0.0:19191          # HTTP endpoint for collecting metrics on the Compactor
+    --objstore.config-file bucket_config.yaml \   # Bucket where compacting will be performed
+    --http-address         0.0.0.0:19191          # HTTP endpoint for collecting metrics on the compactor
 ```
 
-The compactor is not in the critical path of querying or data backup. It can either be run as a periodic batch job or be left running to always compact data as soon as possible. It is recommended to provide 100-300GB of local disk space for data processing.
+Compactor is not in the critical path of querying or data backup. It can either be run as a periodic batch job or be left running to always compact data as soon as possible. It is recommended to provide 100-300GB of local disk space for data processing.
 
-*NOTE: The compactor must be run as a **singleton** and must not run when manually modifying data in the bucket.*
+*NOTE: Compactor must be run as a **singleton** and must not run when manually modifying data in the bucket.*
 
 * *[Example Kubernetes manifest](https://github.com/thanos-io/kube-thanos/blob/master/examples/all/manifests/thanos-compact-statefulSet.yaml)*
 
 [Compactor documentation](components/compact.md)
 
 ## Ruler/Rule
 
-In case of Prometheus with Thanos sidecar does not have enough retention, or if you want to have alerts or recording rules that requires global view, Thanos has just the component for that: the [Ruler](components/rule.md), which does rule and alert evaluation on top of a given Thanos Querier.
+In case Prometheus running with a Thanos sidecar does not have enough retention, or if you want to have alerts or recording rules that require a global view, Thanos has just the component for that: the [Ruler](components/rule.md), which does rule and alert evaluation on top of a given Thanos Querier.
```

Comment:
```suggestion
In case Prometheus running with a Thanos Sidecar does not have enough retention, or if you want to have alerts or recording rules that require a global view, Thanos has just the component for that: the [Ruler](components/rule.md), which does rule and alert evaluation on top of a given Thanos Querier.
```